docs/apicast-crd-reference.md

```diff
@@ -0,0 +1,210 @@
+|                   `staging.externalDnsHostname`                   | `string`  |     Yes      |                   -                   |          DNS hostnames to manage on AWS Route53 by external-dns          |
+|               `staging.loadBalancer.proxyProtocol`                | `string`  |      No      |                  `*`                  | Proxy protocol enabled (k8s aws provider only accepts `*` by the moment) |
+|       `staging.loadBalancer.crossZoneLoadBalancingEnabled`        |  `bool`   |      No      |                `true`                 |      Enable (`true`) or disable (`false`) cross zone load balancing      |
+|         `staging.loadBalancer.connectionDrainingEnabled`          |  `bool`   |      No      |                `true`                 |         Enable (`true`) or disable (`false`) connection draining         |
+|         `staging.loadBalancer.connectionDrainingTimeout`          |   `int`   |      No      |                 `60`                  |                  Connection draining timeout (seconds)                   |
+|   `staging.loadBalancer.connectionHealthcheckHealthyThreshold`    |   `int`   |      No      |                  `2`                  |                 Connection healthcheck healthy threshold                 |
+|  `staging.loadBalancer.connectionHealthcheckUnhealthyThreshold`   |   `int`   |      No      |                  `2`                  |                Connection healthcheck unhealthy threshold                |
+|       `staging.loadBalancer.connectionHealthcheckInterval`        |   `int`   |      No      |                  `5`                  |                Connection healthcheck interval (seconds)                 |
+|        `staging.loadBalancer.connectionHealthcheckTimeout`        |   `int`   |      No      |                  `3`                  |                 Connection healthcheck timeout (seconds)                 |
+|                      `production.image.name`                      | `string`  |      No      | `quay.io/3scale/apicast-cloud-hosted` |                      Image name (docker repository)                      |
+|                      `production.image.tag`                       | `string`  |     Yes      |                   -                   |                                Image tag                                 |
+|                 `production.image.pullSecretName`                 | `string`  |      No      |                   -                   |                 Quay pull secret for private repository                  |
+|                       `production.replicas`                       |   `int`   |      No      |                  `1`                  |                            Number of replicas                            |
+|            `production.env.apicastConfigurationCache`             | `string`  |     Yes      |                   -                   |                     Apicast configurations cache TTL                     |
```

Comment:
Being a field representing time in seconds, shouldnt it be an integer, instead of a string? (applies to examples, docs, CRD)

Reply:
Validations added in 1d9096289bf609153e378ba6ae025774854cbc4f